docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,97 @@
+
+Install the kubectl snap
+---------------------------
+First, install the `kubectl` snap package:
+
+.. code-block:: bash
+				
+  sudo snap install kubectl -–classic
+
+Create a configuration file so that `kubectl` can communicate with MicroK8s by running:
+
+.. code-block:: bash
+				
+  mkdir -p ~/.kube
+  microk8s config > ~/.kube/config
+
+Enable the Intel GPU plugin in your Kubernetes cluster 
+------------------------------------------------------
+
+To ensure DSS can utilise Intel GPUs, you have to enable the Intel GPU plugin in your Kubernetes cluster. The plugin uses the `Intel device plugins for Kubernetes`_ to detect whether an Intel GPU exists on a machine. You can see the `Installation with NFD`_ page for more details.
+
+To enable the plugin, you must download the appropriate `kustomization.yaml` files from Intel’s device plugins for Kubernetes repository, and then apply them on your MicroK8s cluster.
+
+First, build the YAML files for Node Feature Discovery, Node Feature Rules, and the GPU plugin, by running:
+
+.. code-block:: bash
+
+  VERSION=v0.30.0
+  kubectl kustomize https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd?ref=${VERSION} > node_feature_discovery.yaml
+  kubectl kustomize https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd/overlays/node-feature-rules?ref=${VERSION} > node_feature_rules.yaml
+  kubectl kustomize https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/gpu_plugin/overlays/nfd_labeled_nodes?ref=${VERSION} > gpu_plugin.yaml
+
+To allow multiple containers to utilise the same GPU, run:
+
+.. code-block:: bash
+				
+  sed -i 's/enable-monitoring/enable-monitoring\n        - -shared-dev-num=10/' gpu_plugin.yaml
+
+Then, apply the created YAML files to your MicroK8s cluster by running:
+
+.. code-block:: bash
+				
+  kubectl apply -f node_feature_discovery.yaml
+  kubectl apply -f node_feature_rules.yaml
+  kubectl apply -f gpu_plugin.yaml
+
+The MicroK8s cluster should now be configured to recognise and utilise your Intel GPU.
+
+.. note::
+ After the YAML configuration files have been applied, they can be safely deleted.
+
+Finally, 
+
+Verify the DSS CLI detects the GPU
+----------------------------------
+
+Initialise the DSS CLI by running:
+
+.. code-block:: bash
+				
+  dss initialize --kubeconfig=$(microk8s config)
+
+Verify the DSS CLI has detected the GPU by checking the DSS status as follows:
+
+.. code-block:: bash
+
+  dss status
+
+You should expect an output like this:
+
+.. code-block:: bash
+				
+  Output:
+  [INFO] MLflow deployment: Ready
+  [INFO] MLflow URL: http://10.152.183.68:5000
+  [INFO] NVIDIA GPU acceleration: Disabled
+  [INFO] Intel GPU acceleration: Enabled
+
+See also
+--------
+
+* To enable
```

Comment:
```suggestion

```

Reply:
I wanted to add the link to the NVIDIA GPU guide here, but I forgot! Adding it now.

Reply:
@kimwnasptd  I am not sure about this, do we expect an Intel GPU user to have also interest on NVIDIA settings and viceversa?

Reply:
@afgambin yes, in cases where a user might have a machine with Integrated Intel GPU and a dedicated NVIDIA GPU.

So I'd keep it there but with a small restructure of the text (?)

Reply:
We also need to add the link to this one in the NVIDIA page (See also section). @DnPlas

Reply:
I'd say this information is already in the sidebar and we don't need it here. The link we'd be adding here is not to a guide that tells users how to enable both GPUs, or what to do when both GPUs are available, so maybe it is not entirely needed? @afgambin wdyt?

Reply:
I see this is an old comment, and the link has been added actually. Feel free to resolve the discussion. I will not make any changes on this.